0chain/transfer_tokens.sh

```diff
@@ -2,43 +2,35 @@
 
 set -e
 
-echo -e "\n\e[93m===============================================================================================================================================================================
-                                                                        setup variables
-===============================================================================================================================================================================  \e[39m"
-export PROJECT_ROOT=/var/0chain # /var/0chain
-echo -e "\e[32m Successfully Created \e[23m \e[0;37m"
-
 echo -e "\n\e[93m===============================================================================================================================================================================
                                                                         Downloading zwallet binary.
 ===============================================================================================================================================================================  \e[39m"
-pushd ${PROJECT_ROOT} > /dev/null;
-    echo "generating config.yaml file"
-    echo "block_worker: https://mainnnet.zus.network/dns" > config.yaml
-    echo "signature_scheme: bls0chain" >> config.yaml
-    echo "min_submit: 20" >> config.yaml
-    echo "min_confirmation: 20" >> config.yaml
-    echo "confirmation_chain_length: 3" >> config.yaml
-    echo "max_txn_query: 5" >> config.yaml
-    echo "query_sleep_time: 5" >> config.yaml
+echo "Generating config.yaml file"
+echo "block_worker: https://mainnet.zus.network/dns" > config.yaml
+echo "signature_scheme: bls0chain" >> config.yaml
+echo "min_submit: 20" >> config.yaml
+echo "min_confirmation: 5" >> config.yaml
+echo "confirmation_chain_length: 3" >> config.yaml
+echo "max_txn_query: 5" >> config.yaml
+echo "query_sleep_time: 5" >> config.yaml
 
-    if [[ -f bin/zwallet ]] ; then
-        echo "zwallet binary already present"
+if [[ -f bin/zwallet ]] ; then
+    echo "zwallet binary already present"
+else
+    ubuntu_version=$(lsb_release -rs | cut -f1 -d'.')
+    if [[ ${ubuntu_version} -eq 18 ]]; then
+        echo "Ubuntu 18 is not supported"
+        exit 1
+    elif [[ ${ubuntu_version} -eq 20 || ${ubuntu_version} -eq 22 ]]; then
+        curl -L "https://github.com/0chain/zcnwebappscripts/raw/as-deploy/0chain/artifacts/zwallet-binary.zip" -o /tmp/zwallet-binary.zip
```

Comment:
please make sure that this binary is built with latest v1.11.1 tags

Reply:
Nope, this build id of older version. For I will need to test the build on both ubuntu 20, ubuntu22 both.

Reply:
i tested the binary token transfer and staking is working fine